### ISSUE TITLE
Creation and modification times should be UTC

### ISSUE THREAD

**Issue by jeremy-visionaid** [ironfede/openmcdf#161]
According to ["[MS-CFB]: Compound File Binary File Format"](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-cfb/53989ce4-7b05-4f8d-829b-d08d6148375b), creation and modification times should be in UTC, but OpenMcdf uses local time

**Comment by jeremy-visionaid** [comment]
@ironfede I'm happy to create a PR for this, but I have a few others that would conflict with it at the moment

**Comment by jeremy-visionaid** [comment]
@ironfede As I started working on this, I realized that To/FromDateTime is actually fine for the on-disk values since it implicitly converts to UTC already. However, I would expect the DateTimes that OpenMcdf returns to be UTC not local. It's technically a breaking change, but the DateTimes still represent the same times (just `DateTimeKind.Utc` instead of `DateTimeKind.Local`), so clients would most likely be unaffected. Up to you if you want to take this or not or maybe defer till v3.

I've also changed other time measurements to use a monotonic clock (i.e. `Stopwatch`) so that those measurements are unaffected by the system time. I could put that in a separate PR if you don't want the UTC changes right now.
